### PR TITLE
Using Git to retrieve the dropboxingore.

### LINKED ISSUE #5: Installation script should not use wget as a default way to retrieve the dropboxingore
As you can see in the [README.md](https://github.com/sp1thas/dropboxignore/blob/master/README.md) file, there are more than one methods to perform the basic installation. The selected method is not respected by [`install.sh`](https://github.com/sp1thas/dropboxignore/blob/master/utils/install.sh) script because installation script uses `wget` to retrieve the `dropboxignore` shell script.

This should be changed:

 1. `git clone` the whole repo instead of retrieving [`bin/dropboxingore.sh`](https://github.com/sp1thas/dropboxignore/blob/master/bin/dropboxignore.sh) script using `wget`.
 2.  Remove cloned repo after the installation.
 3. Mention on [README.md](https://github.com/sp1thas/dropboxignore/blob/master/README.md) file that `git` should be installed in order to install the package using the basic installation way.

### PR DESCRIPTION
**Pull request by Drallas**
It was a nice challenge and ran a test "works on my machine". 
I hope it's what you where looking for. 

```
$ sudo sh ./install.sh
Cloning into 'dropboxignore'...
remote: Enumerating objects: 87, done.
remote: Counting objects: 100% (87/87), done.
remote: Compressing objects: 100% (62/62), done.
remote: Total 405 (delta 36), reused 59 (delta 16), pack-reused 318
Receiving objects: 100% (405/405), 711.03 KiB | 990.00 KiB/s, done.
Resolving deltas: 100% (195/195), done.
cp bin/dropboxignore.sh /usr/local/bin/dropboxignore
chmod +x /usr/local/bin/dropboxignore
dropboxignore version
dropboxignore: v0.1.4-beta
```

### CODE REVIEW

File: utils/install.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 rm -rf /usr/local/bin/dropboxignore
-wget -q https://raw.githubusercontent.com/sp1thas/dropboxignore/master/bin/dropboxignore.sh -O /usr/local/bin/dropboxignore
-chmod +x /usr/local/bin/dropboxignore
-dropboxignore version
\ No newline at end of file
+git clone https://github.com/sp1thas/dropboxignore.git
+sudo MAKE -C dropboxignore install
```
Comment (sp1thas) [inline utils/install.sh]:
```suggestion
make -C dropboxignore install
```

Comment (sp1thas) [reply utils/install.sh]: `install.sh` script is already executed using `sudo`

File: README.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ This is a simple shell script that can be used to ignore files from dropbox usin
 
 ### Basic Installation
 
-dropboxignore is installed by running one of the following commands in your terminal. You can install this via the command-line with either curl, wget or another similar tool. `attr` package should be installed on your system.
+dropboxignore is installed by running one of the following commands in your terminal. You can install this via the command-line with either curl, wget or another similar tool. `attr` package should be installed on your system. **Git** is required to run the installation.
```
Comment (sp1thas) [inline README.md]:
```suggestion
dropboxignore is installed by running one of the following commands in your terminal. You can install this via the command-line with either curl, wget or another similar tool. `attr` and `git` package should be installed on your system.
```

**Review by sp1thas — CHANGES_REQUESTED** [review]
Great job! please check out some minor improvements.

**Comment by Drallas** [comment]
Thanks for the review; i accepted your changes.

**Review by sp1thas — APPROVED** [review]

**Pull request merged**